### PR TITLE
Remove unnecessary else statements

### PR DESCRIPTION
**Pull request by mortenhauberg**
I removed a bunch of else statements, that doesn't make sense.

I don't know if you guys like that style.
If you do, please just delete this PR :)

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Taylor prefers this for readability.

**Comment by GrahamCampbell** [comment]
Both behaviours are identical so there's no reason to change.

File: src/Illuminate/Console/Parser.php [hunk 1]
```diff
@@ -23,12 +23,11 @@ public static function parse($expression)
 
         preg_match('/[^\s]+/', $expression, $matches);
 
-        if (isset($matches[0])) {
-            $name = $matches[0];
-        } else {
+        if (! isset($matches[0])) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Console/Parser.php]: this change looks good though - could you send it separately please?

File: src/Illuminate/Database/Eloquent/FactoryBuilder.php [hunk 2]
```diff
@@ -103,15 +103,15 @@ public function make(array $attributes = [])
...
         if ($this->amount === 1) {
             return $this->makeInstance($attributes);
-        } else {
-            $results = [];
+        }
 
-            for ($i = 0; $i < $this->amount; $i++) {
-                $results[] = $this->makeInstance($attributes);
-            }
+        $results = [];
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Eloquent/FactoryBuilder.php]: this change looks good though - could you send it separately please?

**Comment by mortenhauberg** [comment]
Ok, i did not know.

I'll make a separate PR for the changes.

Thank you :)

**Pull request closed without merging**